xbmc/Application.cpp

```diff
@@ -252,60 +252,55 @@ using KODI::MESSAGING::HELPERS::DialogResponse;
 //extern IDirectSoundRenderer* m_pAudioDecoder;
 CApplication::CApplication(void)
   : m_pPlayer(new CApplicationPlayer)
+  , m_iScreenSaveLock(0)
```

Comment:
The list is only a nightmare because the class is poorly designed. I find initialization lists more maintainable because the order is enforced, and as a result it's easier to catch variables that go uninitialized.